modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -419,29 +436,28 @@ export default class NestedTabs extends Base {
 	}
 
 	updateIndexValues() {
-		const { $tabContents, $tabTitles } = this.getDefaultElements(),
+		const { $widgetContainer, $tabContents, $tabTitles } = this.getDefaultElements(),
 			settings = this.getSettings(),
-			itemIdBase = $tabTitles[ 0 ].getAttribute( 'id' ).slice( 0, -1 ),
-			containerIdBase = $tabContents[ 0 ].getAttribute( 'id' ).slice( 0, -1 );
+			widgetNumber = $widgetContainer.data( 'widgetNumber' );
 
 		$tabTitles.each( ( index, element ) => {
 			const newIndex = index + 1,
-				updatedTabID = itemIdBase + newIndex,
-				updatedContainerID = containerIdBase + newIndex;
+				updatedTabID = `e-n-tab-title-${ widgetNumber }${ newIndex }`,
+				updatedContainerID = `e-n-tab-content-${ widgetNumber }${ newIndex }`;
 
 			element.setAttribute( 'id', updatedTabID );
 			element.setAttribute( 'style', `--n-tabs-title-order: ${ newIndex }` );
 			element.setAttribute( 'data-tab-index', newIndex );
+			element.setAttribute( 'aria-controls', updatedContainerID );
 
 			element.querySelector( settings.selectors.tabTitleIcon )?.setAttribute( 'data-binding-index', newIndex );
 
 			element.querySelector( settings.selectors.tabTitleText ).setAttribute( 'data-binding-index', newIndex );
-			element.querySelector( settings.selectors.tabTitleText ).setAttribute( 'aria-controls', updatedTabID );
 
-			$tabContents[ index ].setAttribute( 'aria-labelledby', updatedTabID );
-			$tabContents[ index ].setAttribute( 'data-tab-index', updatedTabID );
-			$tabContents[ index ].setAttribute( 'id', updatedContainerID );
-			$tabContents[ index ].setAttribute( 'style', `--n-tabs-title-order: ${ newIndex }` );
+			$tabContents[ index ]?.setAttribute( 'aria-labelledby', updatedTabID );
```

Comment:
In some edge cases, the editor throws an error about the `$tabContents[index]` after using the repeater functions.

By using optional chaining, we avoid our code to crash and for the editor to stop working completely.

Reply:
In which cases does the `$tabContents[ index ]` returns null?

Reply:
In which cases does the $tabContents[ index ] returns null?
Is this a new behavior?